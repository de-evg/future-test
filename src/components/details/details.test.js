import React from 'react';
import renderer from "react-test-renderer";
import Details from "./details";

const props = {address: {streetAddress: "", city: "", state: "", zip: ""}, description: "X", firstName: "X", lastName: "X"};

it(`Render Details correctly`, () => {
  const tree = renderer
    .create(
      <Details details={props} />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
