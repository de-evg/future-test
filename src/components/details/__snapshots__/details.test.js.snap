// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Details correctly 1`] = `
<table
  className="details"
>
  <tbody>
    <tr
      className="details__row"
    >
      <th
        className="details__head"
        colSpan="1"
      >
        Выбран пользователь
      </th>
      <td
        className="details__data"
        colSpan="3"
      >
        X X
      </td>
    </tr>
    <tr
      className="details__row"
    >
      <th
        className="details__head"
        colSpan="1"
      >
        Описание
      </th>
      <td
        className="details__data"
        colSpan="3"
      >
        <textarea
          className="details__description"
          defaultValue="X"
          disabled={true}
        />
      </td>
    </tr>
    <tr
      className="details__row"
    >
      <th
        className="details__head"
        colSpan="1"
      >
        Адрес проживания
      </th>
      <td
        className="details__data"
        colSpan="3"
      >
        
      </td>
    </tr>
    <tr
      className="details__row"
    >
      <th
        className="details__head"
        colSpan="1"
      >
        Город
      </th>
      <td
        className="details__data"
        colSpan="3"
      >
        
      </td>
    </tr>
    <tr
      className="details__row"
    >
      <th
        className="details__head"
        colSpan="1"
      >
        Провинция/штат
      </th>
      <td
        className="details__data"
        colSpan="3"
      >
        
      </td>
    </tr>
    <tr
      className="details__row"
    >
      <th
        className="details__head"
        colSpan="1"
      >
        Индекс
      </th>
      <td
        className="details__data"
        colSpan="3"
      >
        
      </td>
    </tr>
  </tbody>
</table>
`;
